src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java

```diff
@@ -311,6 +314,174 @@ public EmailTemplateVars buildEmailToCafcassAndOtherParties(CaseData caseData) {
             .build();
     }
 
+    private EmailTemplateVars buildEmailTemplateVarsForCitizenWithDashBoardAccess(Map<String, Object> dynamicData) {
+
+        boolean isFinalOrderFlag = dynamicData.get(FINAL).equals(true);
+        boolean multipleOrderFlag = dynamicData.get(MULTIPLE_ORDERS).equals(true);
+        boolean newAndFinalOrderFlag = dynamicData.get(NEW_AND_FINAL).equals(true);
+        boolean languagePreferenceFlag = dynamicData.get(WELSH_EMAIL).equals(true);
+
+        return ManageOrderEmailLip
+            .builder()
+            .emailSubject(buildEmailSubjectForCitizenWithDashBoardAccess(isFinalOrderFlag, multipleOrderFlag,
+                newAndFinalOrderFlag, languagePreferenceFlag, false))
+            .emailTitle(buildEmailSubjectForCitizenWithDashBoardAccess(isFinalOrderFlag, multipleOrderFlag,
+                newAndFinalOrderFlag, languagePreferenceFlag, true))
+            .emailTitleWelsh(languagePreferenceFlag ? buildEmailTitleWelshForCitizenWithDashBoardAccess(isFinalOrderFlag,
+                multipleOrderFlag, newAndFinalOrderFlag) : "")
+            .emailText(buildEmailTextForCitizenWithDashBoardAccess(isFinalOrderFlag, multipleOrderFlag, newAndFinalOrderFlag))
+            .emailTextWelsh(languagePreferenceFlag ? buildEmailTextWelshForCitizenWithDashBoardAccess(isFinalOrderFlag,
+                multipleOrderFlag, newAndFinalOrderFlag) : "")
+            .multipleOrders(multipleOrderFlag || newAndFinalOrderFlag ? "orders" : "order")
+            .multipleOrdersWelsh(languagePreferenceFlag ? multipleOrdersWelsh(multipleOrderFlag, newAndFinalOrderFlag) : "")
+            .caseName(String.valueOf(dynamicData.get("caseName")))
+            .applicantName(String.valueOf(dynamicData.get("name")))
+            .caseLink(String.valueOf(dynamicData.get(DASH_BOARD_LINK)))
+            .caseReference(String.valueOf(dynamicData.get("caseReference")))
+            .build();
+    }
+
+    private String multipleOrdersWelsh(boolean multipleOrderFlag, boolean newAndFinalOrderFlag) {
+
+        if (multipleOrderFlag || newAndFinalOrderFlag) {
+            return "gorchmynion";
+        } else {
+            return "gorchymyn";
+        }
+    }
+
+    private String buildEmailSubjectForCitizenWithDashBoardAccess(boolean isFinalOrderFlag, boolean multipleOrderFlag,
+                                                                  boolean newAndFinalOrderFLag, boolean languagePreferenceFlag,
+                                                                  boolean isTitle) {
+
+        if (!languagePreferenceFlag || isTitle) {
+            if (newAndFinalOrderFLag) {
+                return "New and final court orders issued";
+            } else if (multipleOrderFlag) {
+                if (isFinalOrderFlag) {
```

Comment:
Please move all the static content to new file OrderEmailConstants and follow the below format.
For eg. 
ORDER_NEW_ENG
ORDER_NEW_WEL
ORDER_FINAL_ENG